pipelines/rj_segur__forca_municipal/hash_registry.py

```
@@ -0,0 +1,158 @@
+    """
+    client = _get_bq_client(project_id)
+    table_full = f"{project_id}.{dataset_id}_staging.{registry_table}"
+    query = f"""
+        SELECT 1
+        FROM `{table_full}`
+        WHERE dataset_id   = @dataset_id
+          AND table_id     = @table_id
+          AND id_hash      = @id_hash
+          AND partition_date >= DATE_SUB(CURRENT_DATE(), INTERVAL {HASH_REGISTRY_LOOKBACK_DAYS} DAY)
+        LIMIT 1
+    """
```

Ruff (0.15.12)

[error] Possible SQL injection vector through string-based query construction

(S608)